.github/workflows/autofixes.yml

```diff
@@ -17,32 +20,19 @@ jobs:
         working-directory: e2e-tests
       - run: npm run autofix
       - name: Automated Linting
+        id: automated-linting
         uses: EndBug/add-and-commit@v9
         with:
           message: "Automated linting update and features.json sync"
           add: "*"
           push: true
           pull: --rebase --autostash
           default_author: github_actions
+      - name: Fail on linting push
+        run: |
+          if [[ ${{ steps.automated-linting.outputs.pushed }} == true ]]; then exit 1; else exit 0; fi
```

Comment:
is this check needed? I thought all the workflows would be cancelled automatically if a new commit was made because cancel-in-progress is true in main-workflow

Reply:
It's not completely necessary-- I talked with Oliver and he said that it might be nice to have so that the tests don't even start running.  But yeah, when a new push is made it'll cancel the current tests even if we don't have this.  Are you OK with keeping the check here, or would you rather have it removed?